forge/test/mlir/test_ops.py

```diff
@@ -1909,3 +1909,85 @@ def forward(self, a, b):
     co_out = [co.to("cpu") for co in co_out]
     fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
     assert all([compare_with_golden_pcc(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
+
+
+@pytest.mark.xfail(
+    reason="RuntimeError: Found Unsupported operations while lowering from TTForge to TTIR in forward graph - repeat_interleave"
```

Comment:
Shouldn't this one fail with ttnn.repeat op as required?

Reply:
There is post optimise decomposition for repeat op where it is decomposed to repeat_interleave op. We can remove that too as we asked the support for `ttnn.repeat` op support in MLIR

Reply:
If I understood correctly:
1. torch.repeat == numpy.tile/tvm.tile == forge.repeat == ttnn.repeat?
2. torch.repeat_interleave == numpy.repeat/tvm.repeat == forge.repeat_interleave == ttnn.repeat_interleave?

If yes, let's outline this clearly and provide more detailed explanations in op description. Have in mind that this can be as a Note section of op documentation :))

Reply:
Yes @nvukobratTT . I will add this note in op description as you suggested